### ISSUE TITLE
Script file has always an asterisk "*" before name, doesn't matter if saved

### ISSUE THREAD

**Issue by adamitj** [serge-rider/dbeaver#1484]
This is a little UI bug, does not affect the save feature, but since 4.0.2 version DBeaver do not remove the starting asterisk * before the script name on each script TAB. Even if saved or renamed with CTRL+F2, the asterisk do not disappear.
![captura de tela 2017-03-27 09 56 42](https://cloud.githubusercontent.com/assets/4016463/24357346/c189697c-12d3-11e7-99e0-213013075617.png)

**Comment by serge-rider** [comment]
Please provide steps to reproduce.

**Comment by adamitj** [comment]
Sorry, it was fixed in 4.0.3. I though that I updated an installation of DBeaver 4.0.2 to 4.0.3 but wasn't successful (somehow the application remained open in another user, maybe). Forget about it.

**Comment by adamitj** [comment]
Sorry, but it wasn't fixed. I have this misbehavior on every DBeaver 4.0.3 installations (all updated) on at least 6 different computers running Windows 7, Windows 10, Windows 2008 R2 Server and Windows 2012 Server, all of them 64-bit architecture.

Initially the asterisk does not persists and disappear after pressing CTRL+S or clicking into menu "File, Save". But since this misbehavior starts, do not disappear.

I can't figure out what's happening because the problem starts out of thin air without a different usage of DBeaver tool. Maybe it's a problem only with Windows when the project view has more than 5 files saved - this is the only pattern I saw. 

The problem occurs when create a new script with CTRL+F3 (or clicking on the related button in toolbar) or when opening a saved script from the project view. Even with scripts opened at DBeaver startup.

I was performing regressive tests with older DBeaver versions and could notice that this issue started with version 4.0.2. However, I jumped all my installations from v3.8.2 to v4.0.2 (but my guess is that it started with the very first 4.0.0 version). I always used 64-bit versions.

_Please let this issue open until I find something worth to mention, I'm engaged in doing tests to find a sequence of steps to reproduce._

NOTE: If I remove `**.dbeaver4**` folder inside user default path `%homepath%` the problem disappears for a while.
